src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerRecipes.java

```diff
@@ -0,0 +1,89 @@
+package com.minecolonies.coremod.entity.ai.citizen.baker;
+
+import com.minecolonies.coremod.entity.ai.util.RecipeStorage;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains all recipes the baker can use.
+ */
+public class BakerRecipes
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)